### DISCUSSION TITLE
Site Redirect Error - Too Many Redirects

### DISCUSSION THREAD

**Discussion by jon424** [community/community#130248] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi - I have a domain through GoDaddy. the url is [griftertheband.com](url) I have the following DNS records set: 

![dns_1](https://github.com/community/community/assets/54050695/6b1ed5e2-2b06-4448-a61e-5793a8564574)
![dns_2](https://github.com/community/community/assets/54050695/a741f2d7-410e-4063-8902-2db7bdf87da9)
![dns_3](https://github.com/community/community/assets/54050695/68fdd944-b299-474b-b669-78dc2a977b5c)

I am hosting the site on GitHub pages. Here is what my GitHub Pages settings page looks like. You can see that the DNS check is successful:
![pages-view-github](https://github.com/community/community/assets/54050695/682eadb9-01a8-42b6-98cc-4cafd27b9d2b)

When I try to access the site, I frequently receive a "Too many redirects" error. I have the DNS settings set up (as seen in the screenshots) according to the Custom Domain settings through the GitHub Pages documentation.

Does anyone know why this is happening and how I can prevent this from happening?

here is a link to the repo: https://github.com/jon424/grifter/tree/gh-pages

**Comment by jon424** [comment]
I should add that this happens intermittently - sometimes I get the error, sometimes I can access the site. I would love to understand why it is happening and find out what I can do to make it so that I never see this error. Thanks for any information!

**Reply by yoannchaudet** [reply]
When you are unlucky and DNS resolves to any of these 2 highlighted addresses below, bad things happen:

<img width="932" alt="image" src="https://github.com/user-attachments/assets/3fafec34-d519-47d8-9e2a-9f54f14035ae">

You need to remove these. They are not pointing at GitHub Pages.

Looking at your screenshot above they are marked "cannot edit / cannot remove". There must be an option buried somewhere in GoDaddy to enable HTTP redirection support for your domain. You will want to disable that. This is what is interfering with your site.

**UPDATE**: looking a bit, I assume this is the option you want to turn off https://www.godaddy.com/help/remove-my-domain-forwarding-19979. Once it's off, you will be able to remove the two extra `A` records that should not be there.

**Reply by jon424** [reply]
thanks - I have removed the forwarding and the other domain as well. The site is working now! I basically followed the steps outlined  in the response here: https://stackoverflow.com/questions/78644007/infinite-redirect-between-github-and-godaddy
